Userland/Services/SystemServer/Service.cpp

```diff
@@ -198,10 +198,11 @@ ErrorOr<void> Service::spawn(int socket_fd)
             TRY(Core::System::setenv("SOCKET_TAKEOVER"sv, socket_takeover_builder.string_view(), true));
```

Comment:
(About commit body) Is it possible that there's a typo, or is the sentence too complex for me?

> then only then that will make sense to abort the entire spawn procedure